theforeman.org/pipelines/lib/release.groovy

```diff
@@ -30,7 +30,9 @@ void push_pulpcore_rpms(version, distro) {
 }
 
 void push_katello_rpms(version, distro) {
-    push_rpms("katello-${version}", "katello", version, distro, true)
+    keep_old = version != 'nightly'
+    push_rpms("katello-${version}/katello", "katello/katello", version, distro, keep_old)
+    push_rpms("katello-${version}/candlepin", "katello/candlepin", version, distro, keep_old)
```

Comment:
this will result in things being put into `https://yum.theforeman.org/katello/katello/<version>` and `https://yum.theforeman.org/katello/candlepin/<version>`

not the "nicest" path structure, but predictable.

I could also try using `push_rpms_direct` which allows more fexibility…

Reply:
on the old host, it's `https://fedorapeople.org/groups/katello/releases/yum/<version>/<component>`

Reply:
I do kinda like `<version>/component` given Candlepin is not stand-alone and if you are browsing it gives more of view that you need both paths. If thats harder to do though, this will be fine.